samples/wasi-threads/README.md

```diff
@@ -22,3 +22,11 @@ $ ./iwasm wasm-apps/no_pthread.wasm
 ...
 $ ./iwasm wasm-apps/exception_propagation.wasm
 ```
+
+Run samples in AOT mode (aux stack must be disabled)
+```shell
+$ ../../../wamr-compiler/build/wamrc \
+    --disable-aux-stack-check \
```

Comment:
I wonder if that should be implied when `--enable-multi-thread` is set?

Reply:
That's the thing, it would disable the aux stack not only for WASI threads but also for pthreads. If that's what we want, I'll modify it to use `--enable-multi-thread` and have the aux stack disabled automatically when `--enable-multi-thread` is used. What do you think?